packages/backend/src/open_payments/payment/outgoing/service.test.ts

```diff
@@ -226,22 +227,18 @@ describe('OutgoingPaymentService', (): void => {
   }
 
   beforeAll(async (): Promise<void> => {
-    Config.exchangeRatesUrl = 'https://test.rates'
-    nock(Config.exchangeRatesUrl)
-      .get('/')
-      .query(true)
-      .reply(200, () => ({
-        base: 'USD',
-        rates: {
-          XRP: 0.5
-        }
-      }))
-      .persist()
-    deps = await initIocContainer(Config)
+    const exchangeRatesUrl = 'https://test.rates'
+
+    mockRatesApi(exchangeRatesUrl, () => ({
+      XRP: exchangeRate
+    }))
```

Comment:
This is not the format that we use right now, is it?

Reply:
Those are just for the rates themselves, the actual mocking returns the usual `{ base, rates: { ... }}` object
```ts
export function mockRatesApi(
  exchangeRatesUrl: string,
  getRates: (baseAssetCode: unknown) => Rates['rates']
) {
  return nock(exchangeRatesUrl)
    .get('/')
    .query(true)
    .reply(200, (url) => {
      const base = url.split('=')[1]
      const rates = getRates(base)

      return {
        base,
        rates
      }
    })
    .persist()
}

```

Reply:
Gotcha. Must have missed this with all the changes there were...